third_party/stdlib/re.py

```diff
@@ -0,0 +1,30 @@
+#
+# Secret Labs' Regular Expression Engine
```

Comment:
Oh! Cool okay. Thanks for the explanation. I think we'll probably want to get the code from PyPy in the end because it's pure Python so perhaps it's best if we do a separate PR that pulls in more of `re` from there.